### PR TITLE
Toml-sort

### LINKED ISSUE #43: Format Cargo.toml dependencies alphabetically via toml-sort
### What is the feature you would like to see?

Only a suggestion, opening for discussion first

Using toml-sort makes it significantly easier to manage dependencies by increasing their discoverability in the Cargo.toml. This saves significant when you need to add/remove pallets and therefore add/remove them to each feature list (`std`, `runtime-benchmarks`, `try-runtime`).

Likewise, this would also improve the developer experience for dependency upgrades.

### steps to add
- [ ] add config to repo and run command locally https://github.com/Moonsong-Labs/toml_sort
- [ ] add running the command CI to enforce this formatting

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines

### PR DESCRIPTION
**Pull request by peektism**
Closes #43 

Adds value by enforcing alphabetic order for all dependencies in all Cargo.toml files. Alphabetic ordering of dependencies makes it easier to find them when manually adding/removing/replacing for dependency upgrades.

Completed Tasks
- [x] run toml-sort (all Cargo.toml changes are generated from running `./scripts/toml-sort.sh`)
- [x] add script for running toml-sort which tells user to install it if it's not installed
- [x] incorporate toml-sort into the CI

### CODE REVIEW

File: node/Cargo.toml [hunk 1]
```diff
@@ -1,22 +1,24 @@
 [package]
 name = "parachain-template-node"
-version = "0.1.0"
-authors = ["Anonymous"]
+authors = [ "Anonymous" ]
```
Comment (ozgunozerk) [inline node/Cargo.toml:3]: not a concern for this PR, but should we add our names to here? or mention OpenZeppelin? @orangelash

**Review by ozgunozerk — APPROVED** [review]
LGTM!

File: .github/workflows/cargo-tests.yml [hunk 2]
```diff
@@ -57,6 +57,15 @@ jobs:
       - name: install rust toolchain
         uses: actions-rust-lang/setup-rust-toolchain@v1
 
+      - name: Install toml-sort
+        run: cargo install --git https://github.com/4meta5/toml_sort
```
Comment (KitHat) [inline .github/workflows/cargo-tests.yml:61]: I don't like the dependencies to the personal repositories. We may leave it as it is right now, but lets consider publishing your tool.

**Review by KitHat — APPROVED** [review]

**Comment by ozgunozerk** [comment]
and please use squash-merge

File: .github/workflows/cargo-tests.yml (hunk 2 above)
Comment (KitHat) [reply .github/workflows/cargo-tests.yml:61]: Oooooh
I see.
Then the your approach is correct.
Lets merge it as is and consider two options:
* pushing Moonsong to merge the fix
* taking ownership over the `toml-sort` on OpenZeppelin's behalf 

By the way, we may progress with both things at the same time. Forking to the OpenZeppelin repo and creating an issue along with the fix in the PR back to Moonsong. While it is not merged, we are using our stable git dependency in OZ fork, as it gets merged, we are switching back to the main one.

**Pull request merged**